### ISSUE TITLE
Images are not being displayed

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#2815]
Submitter: taytay
I am using firefox + tampermonkey and the user script

firefox 81.0.1 (64-bit)
tampermonkey 4.11.6117
userscript 1.14.20.4

And the images are not available both in catalog and the threads. In catalogs it just says file name (12312.jpg) while in threads they don't display at all

I disabled all other extensions and scripts, still same

**Comment by jkcdarunday** [comment]
Experienced this too in Fixefox 84.0a1 with Violentmonkey 2.12.7. I looked through the CSS Then I found that werktyme is the cause as it's suddenly enabled (I've never enabled it before) and toggling it off brought back the images. This option suddenly being on could be the same issue that the submitter is experiencing.

Werktym option is on:
![image](https://user-images.githubusercontent.com/4564810/95806472-51688280-0d3a-11eb-8ab7-f6b7e4073eee.png)

Catalog showing just filename:
![image](https://user-images.githubusercontent.com/4564810/95806633-bf14ae80-0d3a-11eb-9cf2-9cc3b85a1c57.png)

Disabling werktyme fixes it:
![Screenshot_20201013_095935](https://user-images.githubusercontent.com/4564810/95806699-deabd700-0d3a-11eb-8aa1-e4559f4510a1.png)
